src/components/storyUtils/AssetMatchingShowcase/AssetMatchingShowcase.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { MatchingAction } from '../../AssetMatching';
+import { uniqueId } from 'lodash';
+import AssetMatching from '../../AssetMatching/Assetmatching';
+import Mocks from './mocks';
+import { formFieldStyles } from '../../../theme/palette';
+
+const AssetMatchingShowcase = ({
+  showCustomContent,
+  styleType,
+}: {
+  showCustomContent: boolean;
+  styleType: formFieldStyles;
+}) => {
+  const [matchingActions, setMatchingActions] = useState<MatchingAction[]>(Mocks.actionsMock);
+
+  const customShowcase = {
+    ...(showCustomContent
+      ? { rightCustomAsset: Mocks.customElement }
+      : { rightAssetProps: Mocks.rightSideData }),
+  };
+
+  const clickHandler = () => {
+    const randomText = uniqueId('action_');
+    const newAction: MatchingAction = {
+      text: randomText,
+      icon: 'actions',
+      onClick: () => {
+        alert(randomText);
+      },
+    };
+
+    setMatchingActions(prevState => [...prevState, newAction]);
+  };
+
+  return (
+    <div>
+      <button css={{ marginBottom: 10, marginRight: 10 }} onClick={clickHandler}>
```

Comment:
rem on those `css`

Reply:
This is just a showcase, it won't affect the component itself. I also check other showcases and none of them uses rem.